README.md

```diff
@@ -1,52 +1,53 @@
-# insilico_wgs
+# insilico\_wgs
 
-You can use this to create bedfiles from a refseq-database or you can do a coverage analysis using mpileup on a collection of validation samples from the novaseq and receive a bunch of output files with various coverage stats. 
+You can use the files in this repo to 
 
-# N
+* create bed files from a refseq-database 
+* do a coverage analysis using mpileup on a collection of validation samples from the novaseq and receive a bunch of output files with various coverage stats.
 
-# create bedfile
 
-./create_bed.py
+### create\_bedfile
```

Comment:
I understand the previous headings better... (Create bed files / Coverage analysis)

Small detail, but: why go down two heading levels each time? I find it looks confusing especially in the part "The genes/transcripts can’t be found" (because "Transcripts" and "Gene names" look so big compared to the section header).